src/typescript/typescript.ts

```diff
@@ -601,6 +579,35 @@ export class TypeScriptProject extends NodeProject {
     }
   }
 
+  /**
+   * Projen default Typescript compiler options.
+   */
+  protected getDefaultTsCompilerOptions(): TypeScriptCompilerOptions {
```

Comment:
Ugh. My bad. Apparently we cannot call this `getFoobar`.... 🤦🏻 

```suggestion
  protected defaultTsCompilerOptions(): TypeScriptCompilerOptions {
```